src/main/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAO.java

```diff
@@ -22,53 +22,76 @@ public interface DarCollectionSummaryDAO extends Transactional<DarCollectionSumm
+        e.election_id, e.status, e.dataset_id, e.reference_id,
+        v.voteid as v_vote_id, dd.dataset_id as dd_datasetid,
+        v.user_id as v_user_id, v.vote as v_vote, v.electionid as v_election_id,
+        v.createdate as v_create_date,v.updatedate as v_update_date, v.type as v_type,
         (regexp_replace(latest_dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'projectTitle' AS name,
         (regexp_replace(latest_dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'status' AS dar_status,
         (regexp_replace(latest_dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'closeoutSupplement' AS closeout,
         ARRAY_AGG(dar_all.reference_id) AS reference_ids
       FROM dar_collection c
-      INNER JOIN users u
-        ON u.user_id = c.create_user_id
+      -- DAR Collection Researcher join
+      INNER JOIN users researcher
+        ON researcher.user_id = c.create_user_id
       LEFT JOIN institution i
-        ON i.institution_id = u.institution_id
+        ON i.institution_id = researcher.institution_id
+      -- DAC User join
+      INNER JOIN users dacUser
+        ON dacUser.user_id = :currentUserId
+      INNER JOIN user_role ur
+        ON dacUser.user_id = ur.user_id AND ur.role_id = :roleId AND ur.dac_id IS NOT NULL
+      INNER JOIN dac dac
+        ON ur.dac_id = dac.dac_id
+      -- Datasets available to DAC
+      INNER JOIN dataset d
+        ON d.dac_id = dac.dac_id
+      -- Restrict DARs to the most recent submission per collection
       INNER JOIN (
         SELECT DISTINCT ON (collection_id) *
         FROM data_access_request
         WHERE submission_date IS NOT NULL
         AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL)
         ORDER BY collection_id, submission_date DESC
         ) latest_dar ON latest_dar.collection_id = c.collection_id
+      -- All DARs for the collection
       INNER JOIN data_access_request dar_all
         ON dar_all.collection_id = c.collection_id
         AND dar_all.submission_date IS NOT NULL
         AND (LOWER(dar_all.data->>'status') != 'archived' OR dar_all.data->>'status' IS NULL)
+      -- Most recent Open and Closed Data Access Elections for DAC User datasets
+      -- Archived, Canceled, and Final elections are not used for status or action calculations
       LEFT JOIN (
         SELECT election.*, MAX(election.election_id) OVER(PARTITION BY election.reference_id, election.dataset_id) AS latest
         FROM election
-        WHERE LOWER(election.election_type) = 'dataaccess' AND election.dataset_id IN (<datasetIds>)
+        WHERE LOWER(election.election_type) = 'dataaccess'
+        AND (LOWER(election.status) = 'open' OR LOWER(election.status) = 'closed')
       ) AS e
         ON e.reference_id = latest_dar.reference_id
+        AND e.dataset_id = d.dataset_id
+      -- Votes for DAC User
       LEFT JOIN vote v
         ON e.election_id = v.electionid
+        AND (LOWER(v.type) = 'final' OR v.user_id = :currentUserId)
```

Comment:
Index on vote.type?

Reply:
Looking into that, thank you!

Reply:
We could use one on election type as well - we use both very frequently in queries.